README.md

```diff
@@ -1,2 +1,33 @@
-# leaves
-Leaves
+# Leaves
+Leaves application makes it easy to bid and search for items whenever and wherever you are. After you log in, you could either enter as a Bidder or as a Planner.
+
+When the user enters as a Planner, they can :
+* Create an event
+* Manage Events
+* Create Items under each event.
+* View reports of their events
+
+As a Bidder, the user is able to browse, search and bid on items.
+
+### Platform
+
+* Android
+
+## Installation
+
+You'll require two things:
+* The [Java jdk](http://support.apple.com/kb/DL1572?viewlocale=en_US&locale=en_US)
+* [Android Studio](http://developer.android.com/sdk/index.html)
+* Clone the project
+* Start Android Studio and open the project
+
+##Tests
+Several tests were written to test the functionality of this application and they are located in the test directory.
+The tests are based on the MOCKITO unit test.
+
+##License
+**Free Software** 
+Simply fork this repository, make your changes and send PR to the STAGING branch.
```

Comment:
If we want to make it a free software, you can look into including any of the open [free software licenses](https://en.wikipedia.org/wiki/Free_software_license)... an example is the popular [MIT license](https://opensource.org/licenses/MIT), which you can easily include.